ui/perfherder/compare/CompareTable.jsx

```diff
@@ -120,11 +118,17 @@ export default class CompareTable extends React.PureComponent {
             </th>
             <th className="table-width-lg">Base</th>
             {/* empty for less than/greater than data */}
-            <th className="table-width-sm" />
+            <th
+              className="table-width-sm"
+              label="empty cell for less than/greater than data"
```

Comment:
I think this should be an `aria-label` from looking at the rule. Please change the text for both labels to remove "empty cell for.." since I think it should act as a descriptive title for the column.

Reply:
Fixed! Thank you